smoker/client/cli.py

```diff
@@ -35,6 +35,86 @@
     'gray' : '\033[0;37m',
 }
 
+
+def _get_plugins():
+    """
+    Get list of available host discovery plugin module names
+    """
+    plugin_dir = os.path.join(os.path.dirname(smoker.client.__file__),
```

Comment:
Did you try to look for some Pythonic plugin managers to use? It would maybe make the code simpler.